### PR TITLE
[fivetv] Add extractor (Closes #5794)

### LINKED ISSUE #5794: 5-tv.ru support
Please add support for 5-tv.ru, videos are available in this pages:
http://5-tv.ru/news/
http://5-tv.ru/video/
http://5-tv.ru/programs/

Examples of urls:
http://5-tv.ru/news/96814/
http://5-tv.ru/news/96781/
http://5-tv.ru/news/96738/
http://5-tv.ru/video/1021729/

### PR DESCRIPTION
**Pull request by hlintala**
Add extractor for 5-tv.ru

### CODE REVIEW

File: youtube_dl/extractor/c5tv.py [hunk 1]
```diff
@@ -0,0 +1,67 @@
+# coding: utf-8
+from __future__ import unicode_literals
+
+from .common import InfoExtractor
+from ..utils import (
+    int_or_none,
+)
+
+
+class C5TVIE(InfoExtractor):
```
Comment (dstftw) [inline youtube_dl/extractor/c5tv.py]: The general naming convention we use is to replace digit with a word it

**Comment by hlintala** [comment]
Apparently naming of C56IE mislead me.

**Comment by dstftw** [comment]
Thanks.

**Comment by dstftw** [comment]
Do you want to be credited as one of the authors of youtube-dl? If so under what name?

**Pull request merged**